MBS-11457: Drop the series ordering_attribute column

reosarevok opened the pull request:
### Implement MBS-11457

This was added back when we thought we would have different attributes to store item numbers for part of series relationships. We don't, so this is always one and the same, 788 (number).

This changes the views to hardcode to 788 and removes the ordering_attribute column elsewhere. It keeps the SERIES_ORDERING_ATTRIBUTE constant on JS since that seems to be used in fields and just points to the 'number' attribute, but removes it on Perl because it was only used to store the ID under ordering_attribute and never use it again, AFAICT.

<!--
    Hello! Thanks for submitting a pull request to MusicBrainz Server.
    We appreciate your time and interest in helping our project!

    Please use this template to help us review your change.

    Depending on your change, some sections may be unneeded, just remove these.
    For example, small pull requests usually don’t need the section “Action”.

    Remember that the more helpful info your pull request includes,
    the easier it is for us to understand and review your changes.

    Ensure that you’ve read through and followed the Contributing Guidelines, at
    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
-->

# Problem
<!--
    Anything that helps us understand why you are making this change goes here.
    What problem are you trying to fix? What does this change address?
-->

:beginner: Reference related tickets with MBS-XXX at least.


# Solution
<!--
    Talk about technical details, considerations, or other interesting points.
-->

:beginner: If you have a lot to say, be more detailed in this section.


# Checklist for author
<!--
    The tasks you have to do to get your change ready for review. Use this if
    you draft a pull request. Mark done tasks with an [x] as you progress. See
    https://github.blog/2019-02-14-introducing-draft-pull-requests/
-->

* [x] Read the [Contributing Guidelines](https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md)
* [ ] Delete irrelevant parts of this template
* [ ] Update the [WikiDocs pages](https://wiki.musicbrainz.org/Category:WikiDocs_Page)
  * [ ] :beginner: Page...
* [ ] :beginner: Task...


# Action
<!--
    Other than merging your change, do you want / need us to do anything else
    with your change? This could include reviewing a specific part of your PR.
-->

1. Make updated WikiDocs pages visible on MusicBrainz website
2. :beginner: Action...

mwiencek commented on admin/sql/updates/20210406-mbs-11457.sql:
```diff
@@ -0,0 +1,81 @@
+
+-- II) Drop the ordering_attribute column
+
+ALTER TABLE series DROP COLUMN ordering_attribute;
```
Unfortunately, there are also sir functions/triggers that reference this column. I think the best course of action here would be to add `CASCADE` and provide instructions to reinstall the sir bits. (We'll need a schema change branch there in any case; tests are currently failing due to that.)

```suggestion
ALTER TABLE series DROP COLUMN ordering_attribute CASCADE;
```

reosarevok replied:
Made the change here - @yvanzo, can you deal with the sir bits?

yvanzo replied:
SIR has been updated in https://github.com/metabrainz/sir/pull/117. It just requires up-to-date `mbdata` models.

mwiencek commented:
@brainzbot test this please

reosarevok commented:
@brainzbot, retest this please

reosarevok commented:
Rebased this on this year's branch, moved from schema 26 to 27, regenerated. Should probably give it a second look anyway, and @yvanzo, we need to look into the sir changes :)

reosarevok commented:
Hmm, actually, I understand the needed sir changes are already merged from last year and just need releasing?

mwiencek commented:
> Hmm, actually, I understand the needed sir changes are already merged from last year and just need releasing?

I think they are already released.  https://github.com/metabrainz/sir/tags says version 2.1.0 included the change to "Stop indexing ordering attribute for series."  Right now we are running v2.1.1 on floyd.

yvanzo reviewed (approved):
LGTMBDNT :ship:
